src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -217,6 +388,25 @@ public class RestartComputerCommand : PSCmdlet, IDisposable
         [Alias("f")]
         public SwitchParameter Force { get; set; }
 
+        /// <summary>
+        /// Specify a delay before the reboot occurs.
+        /// </summary>
+        [Parameter(ParameterSetName = DefaultParameterSet)]
+        [ValidateRange(0, UInt32.MaxValue)]
+        public UInt32 RestartDelay { get; set; }
+    
+        /// <summary>
+        /// Provide a comment for rebooting the computer.
+        /// </summary>
+        [Parameter(ParameterSetName = DefaultParameterSet)]
+        public string Comment { get; set; } = string.Empty;
```

Comment:
Why do we assign empty string? Can the parameter be null? If no we should add validation attribute.

Reply:
😀

Reply:
https://github.com/PowerShell/PowerShell/pull/8112/files#r228037418

It accepts null, will change to nullOrEmpty.

Reply:
Posible do the bot? I lost the bot